cmd/yurthub/app/start.go

```diff
@@ -157,10 +158,28 @@ func Run(cfg *config.YurtHubConfiguration, stopCh <-chan struct{}) error {
 	}
 	trace++
 
-	klog.Infof("%d. new %s server and begin to serve, proxy server: %s, hub server: %s", trace, projectinfo.GetHubName(), cfg.YurtHubProxyServerAddr, cfg.YurtHubServerAddr)
-	s := server.NewYurtHubServer(cfg, certManager, yurtProxyHandler)
+	klog.Infof("%d. create dummy network interface %s and init iptables manager", trace, cfg.HubAgentDummyIfName)
```

Comment:
I think the entire change in this file should be guarded by "if cfg.EnableDummyIf ".    This change should introduce zero overhead if people does not need/disable this feature.

Reply:
fixed